Form-Controls/index.html

```diff
@@ -1,28 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
-
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
-
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>My form exercise</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <header>
+      <h1>Best T-shirts available, book one with us</h1>
+    </header>
+    <main>
+      <form>
+        <fieldset>
+          <label for="name">Name</label>
+          <input type="text" id="name" placeholder="Enter your name" required>
+          <label for="email">Email</label>
+          <input type="email" id="email" placeholder="Email" required>
+          <label for="color">Color</label>
+          <select name="color" id="color" required>
+            <option value="">Select color</option>
+            <option value="red">Red</option>
+            <option value="green">Green</option>
+            <option value="light-green">Light Green</option>
+          </select>
+          <label for="size">Sizes</label>
+          <select name="sizes" id="size" required>
+            <option value="">Select size</option>
+            <option value="XS">XS</option>
+            <option value="S">S</option>
+            <option value="M">M</option>
+            <option value="L">L</option>
+            <option value="XL">XL</option>
+            <option value="XXL">XXL</option>
+          </select>
+          <label for="date">Date</label>
+          <input type="date" id="date" min="2024-11-20">
```

Comment:
I was trying to order in your form but I forgot to set a date and press the submit, Oops it submitted !!! how we can fix this problem?
and another question, did you check [README](https://github.com/CodeYourFuture/Module-User-Focused-Data/blob/main/Form-Controls/README.md)? You can find another requirement for this label.